subscriber_topic: /auv_gnc/trans_ekf/six_dof
publisher_topic: /auv_gnc/controller/thrust

mass: 40.85 # [kg]
center_of_buoyancy: [0.38403, 0.182, -0.1953] # Relative to center of mass [X(m), Y(m), Z(m)]
inertia: [1.32, 2.84, 3.65, 0.0, 0.0] # Inertias [kg-m^2]: Ixx, Iyy, Izz, Ixy, Ixz, Iyz (if products of inertia are negative, specify it) 
linear_drag: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0] # Fdrag = -C*Xdot, Linear drag coeffs for: linear motion XYZ and rotational motion XYZ
quadratic_drag: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0] # Fdrag = -C*sign(Xdot)*Xdot^2, Quadratic drag coeffs for: linear motion XYZ and rotational motion XYZ
thrusters: [ 
  # Pose (from center of mass): [x (m), y (m), z (m), yaw (deg), pitch (deg)]
  # You are allowed to disable thrusters. Do so with care. You are responsible for ensuring
  #   the system is fully controllable.
  {
    id: 0,
    name: "VPF",
    enable: true,
    pose: [0.8255, -0.12065, -0.257175, -135.0, 0.0]
  },
  {
    id: 1,
    name: "VSF",
    enable: true,
    pose: [0.8255, 0.48895, -0.257175, 135.0, 0.0]
  },
  {
    id: 2,
    name: "VPA",
    enable: true,
    pose: [0.03175, -0.12065, -0.257175, -45.0, 0.0]
  },
  {
    id: 3,
    name: "VSA",
    enable: true,
    pose: [0.03175, 0.48895, -0.257175, 45.0, 0.0]
  },
  {
    id: 4,
    name: "HPF",
    enable: true,
    pose: [0.7747, 0, -0.161925, 0.0, 90.0]
  },
  {
    id: 5,
    name: "HSF",
    enable: true,
    pose: [0.7747, 0.3683, -0.161925, 0.0, 90.0]
  },
  {
    id: 6,
    name: "HPA",
    enable: true,
    pose: [0.08255, 0, -0.161925, 0.0, 90.0]
  },
  {
    id: 7,
    name: "HSA",
    enable: true,
    pose: [0.08255, 0.3683, -0.161925, 0.0, 90.0]
  }
]